Automattic/wp-calypso
# Reading Settings: Add Featured image email setting

#### Proposed Changes

* Add Featured image email setting to the new Reading settings page

![Markup on 2022-12-13 at 14:20:24](https://user-images.githubusercontent.com/25105483/207329663-1ffe4b97-d439-445b-b768-4b75da02123c.png)

TODOs:

- [x] Address TS error. Related discussion: p1670575919753569-slack-C02TCEHP3HA; Issue: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/63582; Future task: https://github.com/Automattic/wp-calypso/issues/71054
- [x] Review issue with saving → We will look at this issue separately: https://github.com/Automattic/wp-calypso/issues/71071
- [x] Update the toggle copy

#### Testing Instructions



1. Run local version of Calypso
2. Navigate to `http://calypso.localhost:3000/settings/reading/<YOUR-BLOG>`
3. Ensure the Featured image setting:
  * displays the correct value (enabled / disabled - _default_)
  * can be switched and saved
4. Validate that when the toggle is enabled, the Featured image displays in the _New Post_ email template
5. Test with Simple, Atomic and Jetpack-connected website.

ℹ️ In order for the Featured image to display in the _New Post_ email template, the post needs to have the image attached. To trigger the email, the post needs to be published on a site that has email followers, or [these steps](PdDOJh-jM-p2) can be followed.

ℹ️ Be aware about this issue: https://github.com/Automattic/jetpack/issues/27763.

#### Pre-merge Checklist





Related to #70416
